sdk/src/test/java/com/hcaptcha/sdk/HCaptchaJSInterfaceTest.java

```diff
@@ -116,6 +118,8 @@ public void subset_config_serialization() throws JsonProcessingException, JSONEx
         expected.put("theme", HCaptchaTheme.DARK.toString());
         expected.put("customTheme", JSONObject.NULL);
         expected.put("host", JSONObject.NULL);
+        expected.put("resetOnTimeout", false);
+        expected.put("resetOnError", false);
```

Comment:
`resetOnError` is unused and it should probably be removed.

Reply:
Removed